### DISCUSSION TITLE
mister-quize

### DISCUSSION THREAD

**Discussion by fahdaguenouz** [community/community#140771] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

this is a discution  about mister quize

**Comment by lfarssi** [comment]
when the project will finish?

**Reply by fahdaguenouz** [reply]
yes

**Comment by fahdaguenouz** [comment]
soon we are working on it very hard

**Comment by lfarssi** [comment]
is it responsive thoo

**Comment by fahdaguenouz** [comment]
yes for sure it a web app

**Reply by fahdaguenouz** [reply]
this month will be completed

**Comment by fahdaguenouz** [comment]
hello

**Reply by lfarssi** [reply]
hi there

**Reply by lfarssi** [reply]
are u oky?

**Comment by LiteBrite82** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
